core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -160,9 +152,9 @@ object ConsumerGroupCommand {
     }
 
     protected def describeGroup(group: String) {
-      val configs = parseConfigs(opts)
```

Comment:
I'm not sure I'd expect a breaking change if I was still using the old consumer... I doubt it's a big deal here, but why make the change at all? It doesn't look like the new consumer supports these options, so is there any harm in keeping them in the options?